src/sentry/tasks/groupowner.py

```diff
@@ -77,7 +77,13 @@ def process_suspect_commits(event_id, event_platform, event_frames, group_id, pr
                     if created:
                         owner_count += 1
                         if owner_count > PREFERRED_GROUP_OWNERS:
-                            owners.first().delete()
+                            try:
+                                owner = owners.pop(0)
```

Comment:
No. This means something different. This is a QuerySet, not a list. Why are you trying to do this differently than what I'm suggesting?

Reply:
My point is, [0] translates to a query of `LIMIT 1`. I don't even think `pop()` will work on a QuerySet.

Reply:
<img width="540" alt="image" src="https://user-images.githubusercontent.com/375744/107581259-a8794980-6bac-11eb-9431-b506abccb42d.png">

Reply:
Oops, that's my bad I was thinking this was a list. My primary intention in doing something different was to remove the owner from the list, which I thought owners[0] wouldn't accomplish - but owner.delete() does. Just switching it back to exactly how you said now.